.gitlab-ci.yml

```diff
@@ -376,11 +336,7 @@ test_acceptance_raspberrypi4_bookworm_64bit:
   needs:
     - job: convert_raspbian_raspberrypi4_bookworm_64bit
       artifacts: true
-    - job: convert_raspbian_raspberrypi4_bookworm_32bit
-      artifacts: true
-    - job: convert_raspbian_raspberrypi4_bullseye_64bit
-      artifacts: true
-    - job: convert_raspbian_raspberrypi4_bullseye_32bit
+    - job: convert_raspbian_raspberrypi3_bookworm_64bit
```

Comment:
The loop at line 348 should also include `raspberrypi3_bookworm_64bit_config`, right?